scripts/validate.py

```diff
@@ -124,13 +132,17 @@ def _extract_onboarding_multistage():
         yield from _extract_onboarding()
     elif message_type == "onboarding-multistage":
         yield from _extract_onboarding_multistage()
+    elif message_type == "moments-page":
+        # No actions to validate for moments-page
+        yield None
```

Comment:
I believe you can just do `return` here instead of `yield None`, which makes this function as an empty generator, and the line 144 `if action is not None:` could be deleted too.

Reply:
nice!